cylc/flow/network/schema.py

```diff
@@ -1282,9 +1282,9 @@ def description(self):
 
 
 class BroadcastSetting(GenericScalar):
```

Comment:
Wouldn't making this a `String` type fix the lack of ❔ button?
```suggestion
class BroadcastSetting(String):
```

Reply:
No, the reason the help button isn't showing is because the UI has a special component for broadcast settings which itself does not support the help button, easy fix - https://github.com/cylc/cylc-ui/pull/1058